src/textcode/strings.py

```diff
@@ -198,9 +196,14 @@ def is_relative_path(s):
     """
     Return True if s looks like a relative posix path.
     Example: usr/lib/librt.so.1 or ../usr/lib
+    >>> is_relative_path("../usr/lib")
+    True
+    >>> is_relative_path("../usr")
+    True
+    >>> is_relative_path("..")
+    True
     """
-    relative = re.compile('^(?:([^/]|\.\.)[\w_\-]+/.*$', re.IGNORECASE).match
-    return relative(s)
+    return not os.path.isabs(s)
```

Comment:
IMHO this cannot work as the paths could be POSIX or Windows and you may be running on either one OSes.

Reply:
os.path is either ntpath or posixpath depending on the OS. So this cannot work here since you do not know:
1. which OS will be running scancode
2. for which OS the path found is
You need a cross OS solution. And `os.path` is not it

Reply:
Should I detect which os scancode will be running on and implement separate regular expression pattern for each case?

Reply:
Of course not. The whole point is that scancode should be able to detect a path in a binary that came from any OS and that while running on any OS.